commit 8ca0e12d444906831baca8319ea61b7b5135efd5 (refs/changes/51/51/4)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2016-07-11T19:18:11-07:00 (3 years, 3 months ago)
    
    MANTA-2949 kick_off_pg_transform.js doesn't accept -y
    Reviewed by: Dave Pacheco <dap@joyent.com>
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/bin/kick_off_pg_transform.js b/bin/kick_off_pg_transform.js
index a694bea..467eb21 100755
--- a/bin/kick_off_pg_transform.js
+++ b/bin/kick_off_pg_transform.js
@@ -82,7 +82,7 @@ function parseOptions() {
         // command line, and use the defaults if all else fails.
         var opts = MOLA_CONFIG_OBJ;
         opts.shards = opts.shards || [];
-        var parser = new getopt.BasicParser('a:b:m:no:p:',
+        var parser = new getopt.BasicParser('a:b:m:no:p:y:',
                                             process.argv);
         while ((option = parser.getopt()) !== undefined && !option.error) {
                 switch (option.option) {
